synapse/tests/test_lib_types.py

```diff
@@ -637,3 +644,27 @@ def test_syntag(self):
 
             await self.agenlen(1, core.eval('teststr +:tick=($test, "+- 2day")',
                                             opts={'vars': {'test': '2015'}}))
+
+            await self.agenlen(1, core.eval('teststr +:tick=(now, "-+ 1day")'))
```

Comment:
easy to add them now.

Reply:
to be clear, the above example would not be returned since `2018/12/01` + `+24 hours` is the range `2018/12/01 00:00:00.00` to `2018/12/02 00:00:00.00`; of which `2018/12/02 23:59:59.000` is outside of.